Library/Formula/hugo.rb

```diff
@@ -195,8 +175,10 @@ def install
     ln_sf buildpath, buildpath/"src/github.com/spf13/hugo"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "main.go"
-    bin.install "main" => "hugo"
+    system "go", "build", "-o", bin/"hugo", "main.go"
+
+    bash_completion.mkpath
+    system bin/"hugo", "genautocomplete", "--completionfile=#{bash_completion}/hugo.sh"
```

Comment:
The user _can_ install the completion script anywhere, but I figure if they're installing hugo with Homebrew it ought to be placed in `bash_completion`.